.dockerignore

```
@@ -0,0 +1 @@
+server/node_modules/
```

```suggestion
**/node_modules/
server/config/local*.json
```
The `console` folder also has `node_modules` which should be ignored. The configuration mechanism for the server is the same as for the Display server, so the local files should also be ignored.